www/redirects.yaml

```
@@ -137,3 +137,6 @@
   toPath: /packages/gatsby-cypress/
 - fromPath: /contributing/gatsby-docs-translation-guide/
   toPath: /contributing/translation/
+  # redirecting gatsby-source-filesystem Programmatic Import to plugin README
+  fromPath: /docs/gatsby-source-filesystem-programmatic-import/
```

```suggestion
- fromPath: /docs/gatsby-source-filesystem-programmatic-import/
```
This is missing a character to be valid YAML, that's why it's failing gatsbot.